cmd/waas-auth/main.go

```diff
@@ -40,17 +40,14 @@ func main() {
 		}
 	}
 
-	// HTTP client to use for all outgoing connections out of the enclave
-	client := &http.Client{
-		Timeout: 10 * time.Second,
-		Transport: transport.Chain(
-			baseTransport,
-			transport.SetHeader("User-Agent", "waas-authenticator/"+waasauthenticator.VERSION),
-			traceid.Transport,
-		),
-	}
+	// HTTP transport chain to use for all outgoing connections out of the enclave
+	transportChain := transport.Chain(
+		baseTransport,
```

Comment:
just to confirm: baseTransport has 30s timeout?

Reply:
baseTransport is defined above, the only timeouts it specifies are the idle and TLS handshake. The actual request timeout is defined on the client in `rpc.New`: https://github.com/0xsequence/waas-authenticator/pull/69/files/cd191a960933e9dacc9365af4777e1d85c0671ac#diff-6c518b6e5342273a8ecca52a51a6a7398d6fa4596f53f425212797def8819564R80